paddle/fluid/operators/elementwise/elementwise_add_op.h

```diff
@@ -160,5 +160,31 @@ class ElementwiseAddGradKernel : public ElemwiseGradKernel<T> {
   }
 };
 
+template <typename DeviceContext, typename T>
+class ElementwiseAddDoubleGradKernel : public framework::OpKernel<T> {
+ public:
+  void Compute(const framework::ExecutionContext &ctx) const override {
+    using Tensor = framework::Tensor;
+
+    auto *x = ctx.Input<Tensor>("X");
+    auto *y = ctx.Input<Tensor>("Y");
+    auto *ddx = ctx.Input<Tensor>("DDX");
+    auto *ddy = ctx.Input<Tensor>("DDY");
+
+    auto *ddout = ctx.Output<Tensor>("DDOut");
+
+    // ddOut = ddx + ddy
+    if (ddout) {
+      Tensor ddx_safe, ddy_safe;
+      GetDoubleGradSafeTensor<DeviceContext, T>(ctx, x, ddx, &ddx_safe);
+      GetDoubleGradSafeTensor<DeviceContext, T>(ctx, y, ddy, &ddy_safe);
+
+      ddout->mutable_data<T>(ctx.GetPlace());
+      default_elementwise_add<DeviceContext, T>(ctx, &ddx_safe, &ddy_safe,
+                                                ddout);
+    }
```

Comment:
Whether needs to set  zero for output `framework::GradVarName("X")` and `framework::GradVarName("Y")` ?

Reply:
removed `framework::GradVarName("X")` and `framework::GradVarName("Y")` as output